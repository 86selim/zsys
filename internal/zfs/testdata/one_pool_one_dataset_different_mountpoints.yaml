pools:
  - name: rpool
    datasets:
      - name: .
        mountpoint: /boot
