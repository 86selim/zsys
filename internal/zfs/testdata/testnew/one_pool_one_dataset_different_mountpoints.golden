[
   {
      "Name": "rpool",
      "Mountpoint": "/boot",
      "CanMount": "on"
   }
]